main.py

```diff
@@ -548,11 +569,13 @@ def generate_tracks(cars_to_use, board_to_use, ints_to_use, available_tracks, he
             continue
         if amt_placed_decoy == -1:
             return
-        generate_tracks(list(car_combos[combo_num]), board_to_pass, ints_to_pass,
-                        available_tracks - amt_placed_decoy, np.array(heatmaps), [list(solved[0]), list(solved[1])],
-                        stalled_to_pass, np.array(switch_queue), list(station_stalled),
-                        np.array(crashed_decoys), mvmts_since_solved, available_semaphores - placed_semaphores,
-                        heatmap_limits_pass)
+        yield (
```

Comment:
by using the yield statement you turn a function into a generator, which will run to the next yield statement every time it is iterated. This yields control to the caller, hence the statement name, and is also used to create coroutines, which are the basis of async and await, but that's a few steps of complexity beyond what's happening here